controllers/activemqartemisscaledown_controller_test.go

```diff
@@ -287,6 +287,15 @@ var _ = Describe("Scale down controller", func() {
 			}, existingClusterTimeout, existingClusterInterval).Should(Succeed())
 
 			Expect(k8sClient.Delete(ctx, createdCrd)).Should(Succeed())
+
+			//clean up taint
+			nodes := &corev1.NodeList{}
+			Expect(k8sClient.List(ctx, nodes, &client.ListOptions{})).Should(Succeed())
+			Expect(len(nodes.Items) > 0).Should(BeTrue())
+
+			node := nodes.Items[0]
+			node.Spec.Taints = nil
```

Comment:
there is a revert taint section above, https://github.com/artemiscloud/activemq-artemis-operator/blob/d2270c75113d3e58dba07178b01c93454ace3767/controllers/activemqartemisscaledown_controller_test.go#L262

should that use an nill in place of the emtpy array?

It seems as if the earlier work should be sufficient

Reply:
hmm, I didn't notice that. re-running the test again it seems the taint is properly cleaned up.
I'll close that. Thanks!